src/configure/configure.ts

```diff
@@ -112,6 +144,9 @@ class PipelineConfigurer {
                     if (selectedFolder && selectedFolder.length > 0) {
                         this.workspacePath = selectedFolder[0].fsPath;
                     }
+                    else {
+                        throw new Error(Messages.noWorkSpaceSelectedError);
```

Comment:
do we need to throw error here or can we just exit

Reply:
We not only want to exit from this function, we want to abort everything else after this function call. That's why throw is added here. One way is throw with message operation cancelled or this message.